core/src/main/java/de/javakaffee/web/msm/storage/RedisStorageClient.java

```diff
@@ -178,11 +177,11 @@ public T call() throws Exception {
             try {
                 jedis = _pool.borrowInstance(false);
                 return execute(jedis);
-            } catch (JedisConnectionException e) {
-                // Connection error occurred with this Jedis connection, so now make sure to get a known-good one
-                // The old connection is not given back to the pool since it is defunct anyway
-                if (_log.isDebugEnabled())
-                    _log.debug("Connection error occurred, discarding Jedis connection: " + e.getMessage());
+            } catch (Exception e) {
+                // An error occurred with this Jedis connection, so now make sure to get a known-good one
+                // The old connection is not given back to the pool since it is assumed to be defunct anyway
+                if (_log.isErrorEnabled())
+                    _log.error("Redis connection error occurred, discarding Jedis instance", e);
```

Comment:
The exception is handled and no manual intervention should be needed. Maybe warn would be sufficient?

Reply:
Yes, I can change this if you like.

Reply:
:+1: